airflow/executors/base_executor.py

```diff
@@ -123,9 +123,9 @@ def heartbeat(self):
         self.log.debug("%s in queue", num_queued_tasks)
         self.log.debug("%s open slots", open_slots)
 
-        Stats.gauge('executor.open_slots', open_slots)
-        Stats.gauge('executor.queued_tasks', num_queued_tasks)
-        Stats.gauge('executor.running_tasks', num_running_tasks)
+        Stats.gauge('{}.open_slots'.format(self.__class__.__name__), open_slots)
+        Stats.gauge('{}.queued_tasks'.format(self.__class__.__name__), num_queued_tasks)
+        Stats.gauge('{}.running_tasks'.format(self.__class__.__name__), num_running_tasks)
```

Comment:
we actually probably dont need this one

Reply:
well, I realized that without this our metrics are reported as `Airflow.executor.` rather than `Airflow.CeleryExecutor.` I figured it might be nice to get more specific in case we ever want/need to switch executors.